extensions-core/druid-ranger-security/pom.xml

```diff
@@ -170,10 +184,242 @@
     </dependencies>
     <profiles>
         <profile>
-            <id>hadoop3</id>
+            <id>hadoop2</id>
```

Comment:
Drop that, Hadoop 2 is deprecated and should not be part of this extension

Reply:
Done